app/server.py

```diff
@@ -50,14 +51,18 @@ def _get_bot_data(self, bot_uuid):
                 redis_bot = cloudpickle.loads(redis_bot)
                 bot_data = self._start_bot_process(redis_bot)
                 self._pool[bot_uuid] = bot_data
+                self._set_bot_in_instance_redis(bot_uuid)
             else:
                 print('Creating a new instance...')
-                instance = Bot.get(Bot.uuid == bot_uuid)
+
+                with DATABASE.execution_context():
+                    instance = Bot.get(Bot.uuid == bot_uuid)
```

Comment:
`True` or `False` would be a key on `Bot`?

Reply:
http://docs.peewee-orm.com/en/latest/peewee/querying.html#filtering-records